tljh/requirements-base.txt

```diff
@@ -1,9 +1,10 @@
 # When tljh.installer runs, the users' environment as typically found in
 # /opt/tljh/user, is setup with these packages.
 #
-# FIXME: a frozen version of this file should be used
-# pinning only direct dependencies is a recipe for broken environments!
-
```

Comment:
Oh, I guess I also removed this comment in this PR =/

This was there when all packages were pinned strict.